lib/store_model/combine_errors_strategies/merge_error_strategy.rb

```diff
@@ -7,16 +7,16 @@ module CombineErrorsStrategies
     class MergeErrorStrategy
       # Merges errors on +attribute+ from the child model with parent errors.
       #
-      # @param _attribute [String] name of the validated attribute
+      # @param attribute [String] name of the validated attribute
       # @param base_errors [ActiveModel::Errors] errors object of the parent record
       # @param store_model_errors [ActiveModel::Errors] errors object of the StoreModel::Model
       # attribute
-      def call(_attribute, base_errors, store_model_errors)
+      def call(attribute, base_errors, store_model_errors)
         if Rails::VERSION::MAJOR < 6 || Rails::VERSION::MAJOR == 6 && Rails::VERSION::MINOR.zero?
           base_errors.copy!(store_model_errors)
         else
           store_model_errors.errors.each do |error|
-            base_errors.add(:configuration, :invalid, message: error.full_message)
+            base_errors.add(attribute, :invalid, message: error.full_message)
```

Comment:
this is an ugly bug that was missed by the specs because they lined up exactly 😖